docs/installation.md

```diff
@@ -79,6 +79,10 @@ services:
     environment:
       - TZ=Europe/Berlin # Optional to synchronize the log timestamp with host
     volumes:
+      # Optional to synchronize the log timestamp with host
+      - /etc/localtime:/etc/localtime:ro
+      # Optional to synchronize the hostname with host
+      - /etc/hostname:/etc/hostname:ro
```

Comment:
It will work, but I think it is better to use build in docker feature to pass the hostname to the container (`docker run --hostname xxx`), see https://docs.docker.com/compose/compose-file/compose-file-v3/#domainname-hostname-ipc-mac_address-privileged-read_only-shm_size-stdin_open-tty-user-working_dir for compose

Reply:
Thought of it more like a solution for replicated instances in a swarm environment.

Setting it with the hostname feature would set it to os.Hostname but wouldn't differentiate between instances if more than one is deployed.

The file based solution would always sync the docker node hostname to the container and wouldn't require additional configuration if the deployment replication is modified.

Reply:
os.Hostname is always unique in each container. If you mount "/etc/hostname" and all replicas run on the same host (e.g. docker compose stack), you'll get the same name for each instance.

The goal is to have always a different name per instance.

If you think, there is a use case to sync host name with the host name in container, we can leave this implementation. But we should change the comment/documentation: user should mount "etc/hostname" only if there is only one blocky instance or one instance per host.